src/app/components/Billboard/index.styles.ts

```diff
@@ -5,10 +5,18 @@ export default {
   link: ({ palette }: Theme) =>
     css({
       textDecoration: 'none',
+      color: palette.WHITE,
       '&:hover, &:focus': {
-        color: palette.WHITE,
         textDecoration: 'underline',
       },
+      '::before': {
+        position: 'absolute',
+        top: 0,
+        bottom: 0,
+        left: 0,
+        right: 0,
```

Comment:
Does it still work if we remove these? (If not, leave them, I was more curious than anything!)

Reply:
Clickable area stops working without them annoyingly 🤔

Reply:
Cool, I just wanted to check, thanks for clarifying!